### ISSUE TITLE
Currency style should set approprate default fraction digits precision.

### ISSUE THREAD

**Issue by oscbyspro** [oscbyspro/DiffableTextViews#60]
Default fraction limits should depend on the currency used. It should match values provided by Foundation.NumberFormatter. Below is an example of desirable behavior,

```swift
NumericTextStyle<Decimal>.Currency(code: "USD") ==
NumericTextStyle<Decimal>.Currency(code: "USD").precision(integer: 1..., fraction: 2...2)
```

**Comment by oscbyspro** [comment]
Will be included with 4ddb7b8dda617fbc6725cbfeee2dcbcf4ece92e4.

**Comment by oscbyspro** [comment]
Added in v3.1.0.
